_studio/shared/umc/codec/h264_dec/include/umc_h264_task_supplier.h

```diff
@@ -674,7 +674,7 @@ inline int32_t CalculateDPBSize(uint8_t & level_idc, int32_t width, int32_t heig
 {
     uint32_t dpbSize;
 
-    num_ref_frames = MFX_MIN(16, num_ref_frames);
+    num_ref_frames = std::min<uint32_t>(16, num_ref_frames);
```

Comment:
No need in specification. At least less to change if another type naming is selected.

Reply:
yes